app/src/main/res/values/strings.xml

```diff
@@ -566,26 +566,26 @@
     <string name="dc_custom_block_desc">Coming soon</string>
 
     <string name="dc_dns_website_heading">Show website icon in DNS logs (experimental)</string>
-    <string name="dc_dns_website_desc">Fetches website icons from <i>duckduckgo.com</i>. Consumes up to 350 mb in disk space</string>
+    <string name="dc_dns_website_desc">Fetches website icons from <i>duckduckgo.com</i>. Consumes up to 350 MB of storage space.</string>
 
     <string name="dc_dns_leaks_heading">Prevent DNS leaks</string>
-    <string name="dc_dns_leaks_desc">When enabled, RethinkDNS captures all packets on port 53 and forwards it to a user-set DNS endpoint</string>
+    <string name="dc_dns_leaks_desc">Makes RethinkDNS capture all packets on port 53 and forward them to a user-set DNS endpoint.</string>
```

Comment:
I think the current text string is fine? The period `.` is omitted intentionally from certain strings.

Reply:
The active state when switched on is assumed.